lib/cmds/content-type_cmds/get.js

```diff
@@ -15,6 +15,7 @@ export const builder = (yargs) => {
   return yargs
     .option('id', { type: 'string', demand: true, describe: 'Content Type id' })
     .option('space-id', { type: 'string', describe: 'Space id' })
+    .option('environment-id', { type: 'string', describe: 'Environment id' })
```

Comment:
Can you add `default: 'master'` here to make sure that we always fallback to master

Reply:
👍

Reply:
I will take care of the unit tests

Reply:
Fixed.